pkg/status/dist/templates/forwarder.tmpl

```diff
@@ -7,14 +7,34 @@ Forwarder
 =========
 {{ if .Transactions }}
   {{- range $key, $value := .Transactions }}
+    {{- if and (ne $key "Errors") (ne $key "ErrorsByType")}}
   {{$key}}: {{$value}}
-  {{- end }}
+    {{- end}}
+  {{- end}}
   {{- if .Transactions.DroppedOnInput }}
 
   Warning: the forwarder dropped transactions because all workers were busy.
   You should review your network performance, and tune the forwarder_num_workers
   and forwarder_timeout options.
   {{- end}}
+  {{- if .Transactions.Errors }}
```

Comment:
`.Transactions.Errors` is not incremented on HTTP errors => `HTTPErrorsByCode ` won't show up until you have at least one `ErrorsByType`

Reply:
Actually yes, Transactions.Errors is incremented but only for >400 status code. We can increment it for all error status codes or move http erros on a subcategorie

Reply:
What I meant was that `transactionsErrors.Add(1)` is never called on an HTTP error (anything 4xx or 5xx). Not sure if we understood each-other here.